### PR TITLE
Equipment upgrade Tree view Display

### PR DESCRIPTION
**Pull request by Jebzou**

### CODE REVIEW

File: ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentCraftPlanItemViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using ElectronicObserver.Common;
+using ElectronicObserver.Data;
+using ElectronicObserver.Window.Tools.EquipmentUpgradePlanner.CostCalculation;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Mocks;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
```
Comment (myangelkamikaze) [inline ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentCraftPlanItemViewModel.cs:7]: Add a line of space between the namespace declaration and the class declaration.

Comment (Jebzou) [reply ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentCraftPlanItemViewModel.cs:7]: should be fine now

File: ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/UpgradeTree/UpgradeTreeViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.ObjectModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using ElectronicObserver.Common;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner.UpgradeTree;
+
+public class UpgradeTreeViewModel : WindowViewModelBase
+{
+	public ObservableCollection<UpgradeTreeUpgradePlanViewModel> Items { get; } = new();
```
Comment (myangelkamikaze) [inline ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/UpgradeTree/UpgradeTreeViewModel.cs:9]: This should always be a single item, so you don't need the observable collection.

Comment (myangelkamikaze) [reply ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/UpgradeTree/UpgradeTreeViewModel.cs:9]: Actually, that won't work with TreeView.

**Comment by Jebzou** [comment]
Related to #381

**Pull request merged**